apps/etterlatte-saksbehandling-ui/client/src/components/oppgavebenk/filtreringAvOppgaver/filterLocalStorage.ts

```diff
@@ -1,27 +1,33 @@
 import { initialFilter } from '~components/oppgavebenk/filtreringAvOppgaver/filtrerOppgaver'
 import { logger } from '~utils/logger'
-import { Filter } from '~components/oppgavebenk/filtreringAvOppgaver/typer'
+import { Filter, GosysFilter } from '~components/oppgavebenk/filtreringAvOppgaver/typer'
 
-const FILTER_KEY = 'filter'
+export const FILTER_KEY = 'filter'
+export const GOSYS_FILTER_KEY = 'gosysFilter'
 
-export const leggFilterILocalStorage = (filter: Filter) => localStorage.setItem(FILTER_KEY, JSON.stringify(filter))
+export const leggFilterILocalStorage = (filter: Filter | GosysFilter, filterKey: string) =>
```

Comment:
Ville kanskje hatt to separate funksjoner for dette. Mindre risiko for bruk av feil key og da slipper man å forholde seg til nøklene